src/assets/sass/protocol/components/_call-out.scss

```diff
@@ -0,0 +1,219 @@
+        }
+    }
+}
+
+/* -------------------------------------------------------------------------- */
+// Compact Call Out component
+
+.mzp-c-call-out-compact {
+    background-color: $color-gray-20;
+
+    &.mzp-t-dark {
+        background-color: $color-gray-90;
+        color: $color-white;
+
+        .mzp-c-call-out-desc {
+            color: $color-gray-30;
+        }
+    }
+
+    .mzp-c-call-out-title {
+        @include text-display-sm;
+    }
+
+    .mzp-c-call-out-desc {
+        @include text-body-md;
+    }
+
+    .mzp-c-call-out-cta {
+        margin: $margin-sm 0 0;
+    }
+
+    .mzp-c-button-download {
+        display: inline-block;
+        margin-bottom: 0;
+    }
+
+    &.mzp-t-product-firefox ,
+    &.mzp-t-product-beta,
+    &.mzp-t-product-developer,
+    &.mzp-t-product-nightly,
+    &.mzp-t-product-focus {
+        .mzp-c-call-out-content {
+            @include border-box;
+            @include background-size(64px 64px);
+            @include bidi(((background-position, left top, right top),));
+            background-repeat: no-repeat;
+            padding: (64px + $padding-lg) 0 0 0;
+        }
+    }
+
+    &.mzp-t-product-firefox .mzp-c-call-out-content {
+        @include at2x('#{$image-path}/logos/firefox/firefox.png', 64px, 64px);
+    }
+
+    &.mzp-t-product-beta .mzp-c-call-out-content {
+        @include at2x('#{$image-path}/logos/firefox/beta.png', 64px, 64px);
+    }
+
+    &.mzp-t-product-developer .mzp-c-call-out-content {
+        @include at2x('#{$image-path}/logos/firefox/developer.png', 64px, 64px);
+    }
+
+    &.mzp-t-product-nightly .mzp-c-call-out-content {
+        @include at2x('#{$image-path}/logos/firefox/nightly.png', 64px, 64px);
+    }
+
+    &.mzp-t-product-focus .mzp-c-call-out-content {
+        @include at2x('#{$image-path}/logos/firefox/focus.png', 64px, 64px);
+    }
+
+    @media #{$mq-md} {
+        @include clearfix;
+
+        .mzp-l-content {
+            margin: 0 auto;
+            max-width: 1056px;
+            padding-bottom: $padding-2xl;
+            padding-top: $padding-2xl;
+        }
+
+        &.mzp-t-product-firefox,
+        &.mzp-t-product-beta,
+        &.mzp-t-product-developer,
+        &.mzp-t-product-nightly,
+        &.mzp-t-product-focus {
+            .mzp-c-call-out-content {
+                @include bidi((
+                    (background-position, left center, right center),
+                    (padding, 0 0 0 (64px + $padding-xl), 0 (64px + $padding-xl) 0 0),
+                ));
+                min-height: 64px + ($padding-lg * 2);
+            }
+        }
+
+        .mzp-c-call-out-title,
+        .mzp-c-call-out-desc {
+            margin-bottom: 0;
+        }
+
+        .mzp-c-call-out-title + .mzp-c-call-out-desc {
+            margin-top: $margin-sm;
+        }
+
+        .mzp-c-call-out-content {
+            @include bidi(((float, left, right),));
+            max-width: 624px;
+            width: calc(66% - #{$margin-lg});
```

Comment:
We should try and be consistent with our fractions. I've seen percentages with decimals in other files (33.3%). My personal preference is to use calc(100% * 2/3) instead of 66% so we don't have to deal with fractions of pixels and layout issues

Reply:
I'll update to `66.6%` here, but like I mentioned in the other PR I don't think there's any real benefit in using `calc()` here